packaging/flavorgen/cmd/root.go

```diff
@@ -52,30 +79,74 @@ func RunRoot(command *cobra.Command) error {
 	if err != nil {
 		return errors.Wrapf(err, "error accessing flag %s for command %s", flavorFlag, command.Name())
 	}
+	outputDir, err := command.Flags().GetString(outputDirFlag)
+	if err != nil {
+		return errors.Wrapf(err, "error accessing flag %s for command %s", outputDirFlag, command.Name())
+	}
+	var outputFlavors []string
+	if flavor != "" {
+		outputFlavors = append(outputFlavors, flavor)
+	} else {
+		outputFlavors = allFlavors
+	}
+	generateMultiFlavors := len(outputFlavors) > 1
+	for _, f := range outputFlavors {
+		manifest, err := generateSingle(f)
+		if err != nil {
+			return err
+		}
+
+		fileMapping, ok := flavorMappings[f]
+		if !ok {
+			return fmt.Errorf("file mapping for flavor %q is missng in flavorMappings", f)
+		}
+
+		yamlFileName := fileMapping + ".yaml"
+		if outputDir == "-" {
+			if generateMultiFlavors {
```

Comment:
Would like to keep stdout as an option since the rendered result was output to stdout before the change.
I believe it has been handled in `GenerateManifestYaml` to add the `---` in YAML.

Reply:
That's fine and you're correct. Didn't realize we always add a leading `---`